doc/classes/AudioStreamPlayer.xml

```diff
@@ -44,6 +44,17 @@
 				Plays a sound from the beginning, or the given [param from_position] in seconds.
 			</description>
 		</method>
+		<method name="play_scheduled" experimental="">
+			<return type="void" />
+			<param index="0" name="absolute_time" type="float" />
+			<param index="1" name="from_position" type="float" default="0.0" />
+			<description>
+				Schedules a sound to be played on the [AudioServer]'s timeline at [param absolute_time] in seconds (see [method AudioServer.get_absolute_time]). If [param absolute_time] is in the past, the sound will be played immediately. The sound starts from the given [param from_position] in seconds.
+				Use this method for high precision playbacks, such as a metronome or other rhythm-based sounds.
+				[b]Note:[/b] By default, calling this method will cut off the currently playing sound on this player. Consider changing [member max_polyphony] to prevent this.
+				[b]Note:[/b] On the Web platform, [member playback_type] must be set to [constant AudioServer.PLAYBACK_TYPE_STREAM] for this method to work.
```

Comment:
A glimpse at the code tells me this may be misleading. If it didn't work, nothing would happen.
```suggestion
				[b]Note:[/b] On the Web platform, [member playback_type] must be set to [constant AudioServer.PLAYBACK_TYPE_STREAM] for this method to work. Otherwise, it will behave like [method play].
```
or
```suggestion
				[b]Note:[/b] On the Web platform, this method behaves like [method play], unless [member playback_type] is set to [constant AudioServer.PLAYBACK_TYPE_STREAM].
```
or similar.